### PR TITLE
[UPDATE]Add comment

### PR DESCRIPTION
**Pull request by ChaeyeonSeo**
I add comments and move comments to be in appropriate location.

### CODE REVIEW

**Review by devquint — APPROVED** [review]
I fixed a little bit about commenting convention.

**Comment by Hyuntae-Jeong** [comment]
Fixed position of comment if dictionaries.py

**Review by devquint — CHANGES_REQUESTED** [review]
Commenting convention for this project is writing commenting above the code.
```python
# Description for below
some_code(...)
```
What about reverting latest commit? 

Or you may comment like (bad but simple example)
```python
# After the below code run, a = 5
a=5;
```

File: python-for-beginners/11 - Collections/dictionaries.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
 person = {'first': 'Christopher'}
+
 person['last'] = 'Harrison'
+# person = {'last': 'Harrison', 'first': 'Christopher'}
```
Comment (Hyuntae-Jeong) [inline python-for-beginners/11 - Collections/dictionaries.py:3]: I'm not sure if 'first' comes first or 'last' comes first

Comment (devquint) [reply python-for-beginners/11 - Collections/dictionaries.py:3]: I thinks it just depend upon you.
Python dictionary is unordered, so there's no rule.

Comment (Hyuntae-Jeong) [reply python-for-beginners/11 - Collections/dictionaries.py:3]: Oh cool. I was confused about that.
I will not change the order but add comment about python dictionary
Thanks

**Comment by ChaeyeonSeo** [comment]
I think you don't get "Commenting convention for this project is writing commenting above the code." correctly.

**Review by devquint — APPROVED** [review]

**Review by Dorika0203 — APPROVED** [review]

**Review by Hyuntae-Jeong — APPROVED** [review]

**Review by jesusleejy — APPROVED** [review]

**Pull request merged**